README.md

```diff
@@ -236,6 +236,8 @@ In the Swedish case, changed these are:
 ```
 The new ports are included in the sample `beacon/docker-compose.yml` file. The `/beacon/permissions/auth.py` file (located in `permissions` folder of the `Beacon` repository) needs to be updated with the credentials of the LS AAI. Finally, the `/beacon/permissions/permissions.py` file (located in `permissions` folder of the `Beacon` repository) needs to be changed, by adding a user that should have access to a number of datasets. In the example, the user `<SOME_LS_AAI_USER>` is given access to the test dataset that will be added after starting the services. The `<SOME_LS_AAI_USER>` is the `preferred_username` returned by LS AAI when querying the user-info endpoint.
 
+In order to connect to the Beacon Network, update the `deploy/conf.py` file and specifically the values under the `general`, `project`, `service info` sections, to reflect the information of the organisation, as well as the `Web server configuration` to add the certificates for the TLS connection. Notice that the `uri` should include a `/` in the end, otherwise the URLS from the Beacon Network will not be correct. Once the configuration is done, share the link of the beacon with the Beacon Network people, so that they can add it to the network.
```

Comment:
```suggestion
In order to connect to the Beacon Network, update the `deploy/conf.py` file and specifically the values under the sections `Beacon general info`, `Project info` and `Service` to reflect the information of the organisation. Apart from that, update also the section `Web server configuration` to add the certificates for the TLS connection. Notice that the value of `uri` should include a trailing slash `/`, otherwise the URLs from the Beacon Network will not be correct. Once the configuration is done, share the link of the beacon with the Beacon Network people, so that they can add it to the network.
```

Reply:
We don't provide any email in the previous sections either (i.e. about the LS AAI), so I think I will leave it as is